bitnami/grafana-operator/values.yaml

```diff
@@ -0,0 +1,154 @@
+## ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
+## Example:
+## updateStrategy:
+##  type: RollingUpdate
+##  rollingUpdate:
+##    maxSurge: 25%
+##    maxUnavailable: 25%
+##
+updateStrategy:
+  type: RollingUpdate
+
+image:
+  registry: quay.io
+  repository: integreatly/grafana-operator
+  tag: v3.5.0
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistryKeySecretName
+
+args:
+  ## If one of these options is set a clusterRole and clusterRoleBinding is created to
+  ## ensure that the operator is able to get data from other namespaces.
+  scanAllNamespaces: false
+  scanNamespaces: []
+
+
+grafana:
+  image:
+    registry: docker.io
+    repository: grafana/grafana
+    tag: 7.1.1
+
+grafanaPluginInit:
+  image:
+    registry: quay.io
+    repository: integreatly/grafana_plugins_init
+    tag: 0.0.3
+
+rbac:
+  # create specifies whether to install and use RBAC rules.
+  create: true
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+## Pod securityContext
+## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+##
+securityContext:
+  enabled: true
+  runAsUser: 1001
+  runAsGroup: 1001
+  fsGroup: 1001
+  supplementalGroups: []
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+## Node labels for pod assignment
+## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+##
+nodeSelector: {}
+
+## Tolerations for pod assignment
+## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+##
+tolerations: []
+
+## Affinity for pod assignment
+## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+##
+affinity: {}
+
+## Pod annotations
+## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+##
+podAnnotations: {}
+
```

Comment:
It would be a nice idea to include `commonLabels` and `commonAnnotations`, that will be rendered for all deployed resources:

https://github.com/bitnami/charts/blob/5a5807c1b1db1f2337f6aa5308d3ff73a4329e6a/bitnami/ejbca/values.yaml#L142-L148

Reply:
Already done.